### ISSUE TITLE
Issue updating with apt

### ISSUE THREAD

**Issue by danilofuchs** [grafana/k6#4142]
### Brief summary

Cannot install 0.55.1 through APT

### k6 version

0.55.1

### OS

Pop_OS! 22.04

### Steps to reproduce the problem

```
sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
sudo apt-get update
sudo apt-get install k6
```

### Expected behaviour

Installs

### Actual behaviour

```
Get:1 https://dl.k6.io/deb stable/main amd64 k6 amd64 0.55.1 [28.7 MB]
Err:1 https://dl.k6.io/deb stable/main amd64 k6 amd64 0.55.1
  File has unexpected size (28726870 != 28726866). Mirror sync in progress? [IP: 108.139.182.83 443]
  Hashes of expected file:
   - SHA512:ba485f4455f5203ea6401dcb89efd500002c9bb2951b8b927dd7b2ca504d195358de84c4bfa28f538df836206c01d613d395850f4d0bfd8d167c6a2c270f9874
   - SHA256:2dcaf1223f67366edbd829433ab7fe79af5c7debfa2f37b99fc5afc36d0710c5
   - SHA1:26b043f66299c9904d24c9cd23226c63837aafeb [weak]
   - MD5Sum:eeb70d9835a3c723e3d7ee3bbd21a1fb [weak]
   - Filesize:28726866 [weak]
E: Failed to fetch https://dl.k6.io/deb/dists/stable/main/binary-amd64/k6-v0.55.1-amd64.deb  File has unexpected size (28726870 != 28726866). Mirror sync in progress? [IP: 108.139.182.83 443]
   Hashes of expected file:
    - SHA512:ba485f4455f5203ea6401dcb89efd500002c9bb2951b8b927dd7b2ca504d195358de84c4bfa28f538df836206c01d613d395850f4d0bfd8d167c6a2c270f9874
    - SHA256:2dcaf1223f67366edbd829433ab7fe79af5c7debfa2f37b99fc5afc36d0710c5
    - SHA1:26b043f66299c9904d24c9cd23226c63837aafeb [weak]
    - MD5Sum:eeb70d9835a3c723e3d7ee3bbd21a1fb [weak]
    - Filesize:28726866 [weak]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

**Comment by jcmillett** [comment]
Seeing the same issue when attempting to install k6 during ci/cd pipeline.

**Comment by DavidTolleyAutomatiq** [comment]
This is happening to me in both local and in CI builds, same error

**Comment by leononame** [comment]
On Fedora, we have a similar problem: 
```
>>> Downloading successful, but checksum doesn't match. Calculated: b1e092048e89d121eebd55c8405793016416e2a479bdfdeaec5a1d1de24228a0(sha256)  Expected: 9e57d19cc4b026cf501776b158969d14dc0f421189a431b529600bef88a01b3b(sha256)  - https://dl.k6.io/rpm/x86_64/k6-v0.55.1-amd64.rpm  
```

**Comment by VladKCF** [comment]
We started see k6 installation last night at CI/CD side

**Comment by ankur22** [comment]
Hello everyone, please accept our sincere apologies. It seems that the publish of `v0.55.1` failed. We have released `v0.55.2` which was successful.
